### DISCUSSION TITLE
bug: currency

### DISCUSSION THREAD

**Discussion by BioMedGPH** [Samedis-care/samedis-care-community#44] [Shipped - Live on production]
(not sure if a bug/issue report should go here).

hi, I think something is not going well around the use of currencies. I have set the default currency to NPR (Nepalese Rupees). 

Then, when adding the purchase price for a device I still have to set the currency before I can enter the amount. I would expect it would take the default currency here and only allow change if needed?

Then in the device list the amount I entered is displayed with the EURO symbol. I would expect it takes over the currency set (as default, or as set as deviating currency for that particular device).

**Comment by YvesR** [comment]
You got a point :)
We use the default currency for costs in issue module but you are right, that we can use this default as default for the currency purchase price pull down.

**Reply by BioMedGPH** [reply]
i see the default currency is now shown when adding a purchase price. But I note that in the inventory list still the EUR symbol is used. So our NPR 555.000 device is listed as EUR 555.000.

**Reply by YvesR** [reply]
The inventory do allow different currency, as often (e.g. in doner circumstances) the purchase price is not the facility currency. When you add new inventory the default is now the selected facility currency, nevertheless if you have old inventory records with wrong currency, you need to edit/modify the currency in the record.

**Comment by BioMedGPH** [comment]
hence a mass-edit function would be nice...

But in every inventory (as far as we filled the purchase price) the 
currency is set to Nepali Rupee. But in the Device/inventory list it 
shows as EUR

from the device inventory item:


from the inventory list view:

This is for all our devices. I see no way actually to change this? It is 
just that the list view displays the wrong currency symbol.

Arjan

..................
Bio-Medical Engineering department
INF Nepal, Green Pastures Hospital
Hariyokharka,Pokhara
